pandas/_libs/groupby.pyi

```diff
@@ -84,11 +86,12 @@ def group_ohlc(
     min_count: int = ...,
 ) -> None: ...
 def group_quantile(
-    out: np.ndarray,  # ndarray[float64_t, ndim=2]
+    out: npt.NDArray[np.float64],
     values: np.ndarray,  # ndarray[numeric, ndim=1]
-    labels: np.ndarray,  # ndarray[int64_t]
-    mask: np.ndarray,  # ndarray[uint8_t]
-    qs: np.ndarray,  # const float64_t[:]
+    labels: npt.NDArray[np.intp],
+    mask: npt.NDArray[np.uint8],
+    sort_arr: npt.NDArray[np.intp],  # const
```

Comment:
i think we call this sorter elsewhere (or maybe indexer); just want to be consistent

Reply:
Sounds good, have renamed to `sort_indexer`, which is more explicit and more consistent with usage in `algos.pyx`